docs/.vuepress/config.js

```diff
@@ -72,6 +72,10 @@ module.exports = {
           {
             text: 'Cookbook',
             link: '/cookbook/'
+          },
+          {
+            text: 'Migration',
+            link: '/guide/upgrading/'
```

Comment:
404 here: https://a3.docs.apos.dev/guide/upgrading/

Reply:
Also it would be good to organize this list so it matches the order of the list on the docs index page.